src/flashbar/styles.scss

```diff
@@ -66,22 +66,21 @@
   display: flex;
   flex-grow: 1;
   min-width: 0;
+}
```

Comment:
This fixes an existing bug I noticed: the focus ring in the Flash element was not being displayed correctly because the corresponding CSS rules were inside the selector for `.flash-body` instead of `.flash-focus-container`